examples/bin/start-druid-main

```diff
@@ -0,0 +1,436 @@
+}
+
+SERVICE_MEMORY_HEAP_PERCENTAGE = {
+    MM: 1,
+    ROUTER: 1,
+    COORDINATOR: 1,
+    BROKER: 0.60,
+    HISTORICAL: 0.40,
+    MM_TASK: 0.50
+}
+
+LOGGING_ENABLED = False
+
+def custom_print(message):
+    if LOGGING_ENABLED:
+        print(message)
+
+def error_and_exit(message):
+    sys.stderr.write(message + '\n')
+    sys.exit(1)
+
+def configure_parser():
+    parser = argparse.ArgumentParser(
+        prog='Druid quickstart',
+        formatter_class=argparse.RawTextHelpFormatter,
+        epilog=
+"""
+sample usage:
+    start-druid
+            Start up all the services (including zk).
+            50 - 80 percent of system memory is used.
+    start-druid -m=100g
+            Start up all the services (including zk)
+            using the given memory.
+    start-druid -m=100g --compute_only
+            Compute memory distribution and validate
+            arguments.
+    start-druid -m=100g -sl=broker,router
+            Start broker & router service, using 100g of memory.
+            Read config from conf/druid/single-server/quickstart.
+    start-druid -m=100g --sl=broker,router \\
+    -cp=conf/druid/single-server/custom
+            Start broker & router service, using 100g of memory.
+            Read config from <config_path>.
+            Since <memory> is specified, exception is thrown if
+            jvm.config is present for any of the services.
+    start-druid -sl=broker,router \\
+    -cp=conf/druid/single-server/custom
+            Start broker & router service, using system memory.
+            If jvm.config is specified for both the
+            services within <config_path>/<service>,
+            memory distribution is not calculated.
+            If jvm.config is present for either of the services,
+            exception is thrown.
+            If jvm.config is not present for both of the services,
+            memory distribution is calculated.
+    start-druid -m=100g \\
+    -cp=conf/druid/single-server/custom \\
+    -sl=broker,router \\
+    --run_zk
+            Start broker, router and zookeeper.
+            zk config is read from conf/zk.
+"""
+    )
+    parser.add_argument('--memory', '-m', type=str, required=False,
+                        help='Total memory for all processes (services and tasks, if any). \n'
+                             'This parameter is ignored if each service already has a jvm.config \n'
+                             'in the given conf directory. e.g. 500m, 4g, 6g\n')
+    parser.add_argument('--service_list', '-sl', type=str, required=False,
+                        help='List of services to be started, subset of \n'
+                             '{broker, router, middleManager, historical, coordinator-overlord}. \n'
+                             'If the argument is not given, all services \n'
+                             'and zookeeper is started. e.g. -sl=broker,historical')
+    parser.add_argument('--config_path', '-cp', type=str, required=False,
```

Comment:
Nit: Maybe rename this param to `--configs` or `--config`

Reply:
I am using `--config`.